src/sanitizer.js

```diff
@@ -157,6 +161,8 @@ export function sanitizeHtml(html) {
           for (let i = 0; i < attribs.length; i += 2) {
             if (WHITELISTED_ATTRS.indexOf(attribs[i]) != -1) {
               attribs[i + 1] = savedAttribs[i + 1];
+            } else if (attribs[i].search(WHITELISTED_ATTR_PREFIX_REGEX) == 0) {
```

Comment:
I am not entirely sure this needs to be a Regex, but thought might be useful in the future if we need to whitelist more prefixes. However, happy to change this to just a simple `indexOf('data-') == 0` check instead for speed. LMK.